src/test/java/com/michalsydoryk/app/boardchecker/HorizontalCheckerTest.java

```diff
@@ -0,0 +1,50 @@
+package com.michalsydoryk.app.boardchecker;
+
+import com.michalsydoryk.app.board.Board;
+import com.michalsydoryk.app.board.Board2D;
+import com.michalsydoryk.app.coordinates.Coordinates2D;
+import com.michalsydoryk.app.sign.Sign;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+@Test
+public class HorizontalCheckerTest {
+    private static final Sign signForTests = Sign.CROSS;
+    private static final int boardSize = 30;
+    private static final int combinationSize = 5;
+    private static final int x = 15;
+    private static final int y = 15;
```

Comment:
Constants should be uppercase, but these don't need to be constants I think.

Reply:
i miss that as many things. thx